backend/endpoints/rom.py

```diff
@@ -216,6 +216,15 @@ async def update_rom(
     cleaned_data["summary"] = data.get("summary", db_rom.summary)
     cleaned_data["url_cover"] = data.get("url_cover", db_rom.url_cover)
     cleaned_data["url_screenshots"] = json.loads(data["url_screenshots"])
+    cleaned_data["total_rating"] = data.get("total_rating", db_rom.total_rating)
+    cleaned_data["genres"] = json.loads(data["genres"])
+    cleaned_data["franchises"] = json.loads(data["franchises"])
+    cleaned_data["collections"] = json.loads(data["collections"])
+    cleaned_data["expansions"] = json.loads(data["expansions"])
+    cleaned_data["dlcs"] = json.loads(data["dlcs"])
+    cleaned_data["remakes"] = json.loads(data["remakes"])
+    cleaned_data["companies"] = json.loads(data["companies"])
+    cleaned_data["first_release_date"] = data.get("first_release_date", db_rom.first_release_date) or None
```

Comment:
at some point we should consider just passing an igdb_id from search, and re-fetching the info on the backend

Reply:
Totally agree, this is starting to becoming a nightmare if we keep adding metadata. Not in this PR because I have plans to refactor the igdb_handler, but I noticed this problem too

Reply:
ok maybe you can take over the mobygames branch and make those changes there (after i rebase it)